src/Abp/BackgroundJobs/AsyncBackgroundJob.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading.Tasks;
+using Abp.Events.Bus;
+
+namespace Abp.BackgroundJobs
+{
+    public abstract class AsyncBackgroundJob<T> : BackgroundJob<T>
+    {
+        public override void Execute(T args)
+        {
+            ExecuteAsync(args).Wait();
```

Comment:
Instead, use AsyncHelper:

```suggestion
            AsyncHelper.RunSync(() => ExecuteAsync(args));
```